src/objects/zcl_abapgit_oo_base.clas.abap

```diff
@@ -313,4 +319,35 @@ CLASS ZCL_ABAPGIT_OO_BASE IMPLEMENTATION.
     DELETE FROM seocompotx WHERE clsname = is_key-clsname. "#EC CI_SUBRC
     INSERT seocompotx FROM TABLE it_descriptions.         "#EC CI_SUBRC
   ENDMETHOD.
+
+  METHOD get_force_method_order.
+    DATA: lv_object_package TYPE devclass,
+          lt_packages       TYPE zif_abapgit_sap_package=>ty_devclass_tt,
+          lv_obj_name       TYPE sobj_name,
+          lo_repo           TYPE REF TO zcl_abapgit_repo.
+
+    lv_obj_name = iv_clsname.
+    rv_force_method_order = abap_false.
+    ##TODO.
+    " get_repo_by_object or get_repo_by_package would be nice...
+    TRY.
```

Comment:
this method seems strange/misplaced, and I guess it will impact performance on all CLAS serialization?

Reply:
Yea, I dislike it too.
Do you have a suggestion on how to pass the flag through to serialize_abap_clif_source? Just add a parameter through the whole callstack? Or add another constructor parameter for CLAS and special handling in ZCL_ABAPGIT_OBJECTS->CREATE_OBJECT ?

Reply:
Hmm, I moved stuff around a bit. I think it's better than before, but not sure if acceptable.